ports/perfetto/CMakeLists.txt

```
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.8.0)
+project(perfetto)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(CMakePackageConfigHelpers)
+
+include_directories(sdk)
+add_library(perfetto sdk/perfetto.cc)
```

I think we still need
```suggestion
add_library(perfetto sdk/perfetto.cc)
target_include_directories(perfetto PUBLIC $<LINK_INTERFACE:include>)
```
~~~

---

Feedback should all be addressed.